package.json

```diff
@@ -28,5 +28,8 @@
     "lint": "tslint -c tslint.full.json --project tsconfig.json --type-check --exclude 'examples/**/*' --exclude '**/node_modules/**' --exclude '**/*.d.ts' '**/*.ts'",
     "test": "mocha --opts test/mocha.opts.root 'packages/*/test/**/*.ts'",
     "posttest": "npm run lint"
+  },
+  "dependencies": {
+    "reflect-metadata": "^0.1.10"
```

Comment:
Yes, it _can_.. But that is only really viable with `devDependencies`. With regular dependencies you'll end up publishing a broken `@loopback/authenticate` because it won't have its own.